addons/project/static/src/views/project_project_kanban/project_project_kanban_header.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module */
+
+import { KanbanRenderer } from "@web/views/kanban/kanban_renderer";
+import { KanbanHeader } from "@web/views/kanban/kanban_header";
+import { useService } from "@web/core/utils/hooks";
+
+export class ProjectProjectKanbanRenderer extends KanbanRenderer {}
+
+export class ProjectProjectKanbanHeader extends KanbanHeader {
```

Comment:
Could you keep two separate files for those two classes, like we do for ProjectTaskKanbanRenderer?